RecoCTPPS/TotemRPLocal/src/TotemTimingRecHitProducerAlgorithm.cc

```diff
@@ -17,7 +17,7 @@ const float TotemTimingRecHitProducerAlgorithm::SINC_COEFFICIENT = std::acos(-1)
 
 TotemTimingRecHitProducerAlgorithm::TotemTimingRecHitProducerAlgorithm(
     const edm::ParameterSet &iConfig)
-    : sampicConversions_(iConfig.getParameter<std::string>("calibrationFile")),
+    : sampicConversions_(TotemTimingConversions(iConfig.getParameter<bool>("mergeTimePeaks"), iConfig.getParameter<std::string>("calibrationFile"))),
```

Comment:
Constructor + copy; I would suggest you to stick to the previous version with an extra argument